src/pages/briefs/[id]/apply.tsx

```diff
@@ -180,15 +180,15 @@ export const SubmitProposal = (): JSX.Element => {
   const milestoneAmountsAndNamesHaveValue = allAmountAndNamesHaveValue();
 
   return (
-    <div className="flex flex-col gap-[2.5rem] text-base leading-[1.5] hq-layout">
+    <div className="flex flex-col gap-10 text-base leading-[1.5] hq-layout">
       <div>
-        <h3 className="ml-[2rem] mb-[0.5rem] text-xl leading-[1.5] font-bold m-0 p-0  flex">
+        <h3 className="ml-8 mb-2 text-xl leading-[1.5] font-bold m-0 p-0  flex">
           Job description
         </h3>
         {brief && <BriefInsights brief={brief} />}
       </div>
       <div>
-        <div className="milestones border border-solid border-[#fff] py-[20px] rounded-[20px] bg-theme-grey-dark">
+        <div className="milestones border border-solidborder-white py-5 rounded-[20px] bg-theme-grey-dark">
```

Comment:
The space got removed between border-solif and border white. I think border-solid doesn't effect much so it's safe to remove border-solid and we are using border-light-white for thinner border look.